google/cloud/bigquery/v2/minimal/internal/common_v2_resources.cc

```diff
@@ -139,9 +129,8 @@ void from_json(nlohmann::json const& j, StandardSqlDataType& t) {
     auto const index = j.at("sub_type_index").get<int>();
```

Comment:
(Don't fix in this PR, just an FYI):

This is another double look up. `j.contains(...)` followed by `j.at(...)`.